src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -1,33 +1,93 @@
-import * as React from "react";
-
+import React, { useState } from "react";
 import Checkbox, { CheckboxProps } from "./Checkbox";
 import { Story } from "@storybook/react/types-6-0";
 
+// Set defaults for the checkbox stories.
 export default {
     title: "Checkbox",
     component: Checkbox,
+    // Remove some props for all checkbox stories.
+    argTypes: {
+        checkboxId: {
+            table: {
+                disable: true,
+            },
+        },
+        onChange: {
+            table: {
+                disable: true,
+            },
+        },
+        labelOptions: {
+            table: {
+                disable: true,
+            },
+        },
+    },
 };
 
 const Template: Story<CheckboxProps> = (args) => <Checkbox {...args} />;
 
-export const BasicCheckbox = Template.bind({});
-BasicCheckbox.args = {
+// Uncontrolled checkbox.
+export const UncontrolledCheckbox = Template.bind({});
+
+// Uncontrolled checkbox set props
+UncontrolledCheckbox.args = {
     checkboxId: "checkbox",
-    isSelected: false,
+    defaultChecked: false,
     labelOptions: {
         id: "label",
-        labelContent: <>Label Text</>,
+        labelContent: <>This is an uncontrolled checkbox.</>,
     },
     onChange: () => {},
 };
 
-export const CheckboxWithLongLabel = Template.bind({});
-CheckboxWithLongLabel.args = {
-    checkboxId: "checkbox2",
-    isSelected: true,
-    labelOptions: {
-        id: "label",
-        labelContent: <>This is a long label but just another example</>,
+// Uncontrolled checkbox set controls
+UncontrolledCheckbox.argTypes = {
+    checked: {
+        table: {
+            disable: true,
+        },
+    },
+};
+
+// Controlled checkbox.
+const ControlledCheckboxTemplate: Story<CheckboxProps> = (args) => {
+    const [isChecked, setChecked] = useState<boolean>(false);
+
+    const onChange = (event) => {
+        setChecked(!isChecked);
+    };
+
+    return (
+        <Checkbox
+            {...args}
```

Comment:
what additional args are being passed?  Can we pass them directly instead of in a spread like this?  Because this is documentation, I'd like these things to be pretty explicit about what is and is not expected.

Reply:
I think just checked is needed. Most of the examples from the storybook documentation have the args spread: https://storybook.js.org/docs/react/essentials/controls -- thats where I got this from. I'm fine with doing either. Ive made some changes to get this to work with controls, so there's also this in the storybook example now that explains what we're doing with the args:

```javascript
// Allows storybook controls to toggle checked state.
useEffect(() => {
  setChecked(args.checked);
}, [args.checked]);
```